base/docs/helpdb.jl

```diff
@@ -7211,9 +7211,9 @@ Given `@osx? a : b`, do `a` on OS X and `b` elsewhere. See documentation for Han
 :@osx
 
 doc"""
-    ishermitian(A) -> Bool
+    ishermitian(A; tol=eps(eltype(real(A)))*norm(A,1)) -> Bool
```

Comment:
I think you mean `real(eltype(A))`.  Doing `real(A)` makes a real copy of the matrix if it is complex.